For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2162 -- read without -r will mangle backs...
  https://www.shellcheck.net/wiki/SC2185 -- Some finds don't have a default p...

--- a/ownersort.sh
+++ b/ownersort.sh
@@ -1,5 +1,6 @@
-#!/bin/bash
-find -user $USER|while read i
+#!/bin/env bash
+find . -user "$USER"|while read -r i
 do
-  cp $i /home/$USER/$i
+  cp "$i" /home/"$USER"/"$i"
 done
+

